[14.0][ADD]sale_purchase_requisition (#2430)
OCA/sale-workflow: sale_purchase_requisition/readme/DESCRIPTION.rst

This module connects a quotation with a purchase agreement.

> A smart button is added in the quotation and in the purchase agreement.

Reviewer: ```suggestion
Smart buttons are added in the quotation and in the purchase agreement.
```

> Automatically, all the quotation lines are passed to the purchase agreement and the person in charge will be the purchase salesperson

Reviewer: ```suggestion
Automatically, all the quotation lines are passed to the purchase agreement and the person in charge will be the purchase agent.
```
